Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/README.md

```diff
@@ -738,20 +733,91 @@ Lists details about all member accounts for the current Security Hub master acco
-**No entries.**
+#### Human Readable Output
+
+>### AWS SecurityHub ListMembers
+>**No entries.**
+
+
+### get-remote-data
+***
+Get remote data from a remote incident. This method does not update the current incident, and should be used for debugging purposes only.
+
+
+#### Base Command
+
+`get-remote-data`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| id | The remote incident ID. | Required | 
+| lastUpdate | The UTC timestamp in seconds of the last update. The incident is only updated if it was modified after the last update time. Default is 0. | Optional | 
+
+
+#### Context Output
+
+There is no context output for this command.
+### get-mapping-fields
+***
+Returns the list of fields to map in outgoing mirroring. This command is only used for debugging purposes.
+
+
+#### Base Command
+
+`get-mapping-fields`
+#### Input
+
+There are no input arguments for this command.
+
+#### Context Output
+
+There is no context output for this command.
+
+## Incident Mirroring
+
+You can enable incident mirroring between Cortex XSOAR incidents and AWS - Security Hub corresponding events (available from Cortex XSOAR version 6.0.0).
+To set up the mirroring:
+1. Enable *Fetching incidents* in your instance configuration.
+2. In the *Mirroring Direction* integration parameter, select in which direction the incidents should be mirrored:
+
+    | **Option** | **Description** |
+    | --- | --- |
+    | None | Turns off incident mirroring. |
+    | Incoming | Any changes in AWS - Security Hub events (mirroring incoming fields) will be reflected in Cortex XSOAR incidents. |
+    | Outgoing | Any changes in Cortex XSOAR incidents will be reflected in AWS - Security Hub events (outgoing mirrored fields). |
+    | Incoming And Outgoing | Changes in Cortex XSOAR incidents and AWS - Security Hub events will be reflected in both directions. |
+
+
+Newly fetched incidents will be mirrored in the chosen direction. However, this selection does not affect existing incidents.
+**Important Note:** To ensure the mirroring works as expected, mappers are required, both for incoming and outgoing, to map the expected fields in Cortex XSOAR and AWS - Security Hub.
+
+
+### Mirroring In Notes
+The mirroring in works when the security-findings provider last updated the finding record. 
+If a user updated an incident it won't show in the XSOAR incident.
+
+### Mirroring Out Notes
+The supported fields in the mirroring out process are:
+1. AWS Security Hub Confidence.
+2. Risk Score
+3. Comment
+4. Severity
+5. AWS Security Hub Verification State
+6. AWS Security Hub Workflow Status
+
+**Important Note:** that the mirror out updates do not affect the value of UpdatedAt for a finding. These updates can be overwritten by the security-findings provider.
 
+* Note: Before updating the comment, make sure that you chose an owner to the incident, in the Case Details section. Otherwise, the comment update won't work.
```

Comment:
```suggestion
* Before updating a comment, make sure that you chose an owner to the incident in the Case Details section. Otherwise, the comment update won't work.
```